adapter/memQuota/memQuota.go

```diff
@@ -150,21 +155,16 @@ func (mq *memQuota) Close() error {
 	return nil
 }
 
-// getCurrentTicks returns the number of ticks since January 1st 1970 UTC
-func getCurrentTicks() int64 {
-	return time.Now().UnixNano() / nanosPerTick
-}
-
-func (mq *memQuota) Alloc(args adapter.QuotaArgs) (int64, error) {
+func (mq *memQuota) Alloc(args adapter.QuotaArgs) (int64, time.Duration, error) {
```

Comment:
Yeah, I was debating whether it was worth it. I guess it is, it would also be symmetric with the input and how I handle this elsewhere.